amethyst_animation/src/resources.rs

```diff
@@ -42,15 +33,11 @@ pub trait AnimationSampling: Send + Sync + 'static + for<'b> ApplyData<'b> {
         &mut self,
         channel: &Self::Channel,
         data: &Self::Primitive,
-        extra: &<Self as ApplyData<'a>>::ApplyData,
+        buffer: &mut CommandBuffer,
```

Comment:
Should we enforce the same naming schema for all CommandBuffers in fn arguments inside of amethyst?

Reply:
I only see one instance of it being named differently, in the removal util.  I think it would be good to stay consistent but I don't know of a linting tool to enforce it.  Do you have something in mind?